src/OrchardCore.Modules/OrchardCore.Contents/Services/DefaultContentsAdminListQueryService.cs

```diff
@@ -1,45 +1,128 @@
             IServiceProvider serviceProvider,
             IEnumerable<IContentsAdminListFilter> contentsAdminListFilters,
+            IOptions<ContentsAdminListFilterOptions> contentsAdminListFilterOptions,
             ILogger<DefaultContentsAdminListQueryService> logger)
         {
             _session = session;
             _serviceProvider = serviceProvider;
             _contentsAdminListFilters = contentsAdminListFilters;
+            _contentsAdminListFilterOptions = contentsAdminListFilterOptions.Value;
             _logger = logger;
         }
 
         public async Task<IQuery<ContentItem>> QueryAsync(ContentOptionsViewModel model, IUpdateModel updater)
         {
-            // Because admin filters can add a different index to the query this must be added as a Query<ContentItem>()
+            var defaultTermNode = model.FilterResult.OfType<DefaultTermNode>().FirstOrDefault();
+            var defaultTermName = defaultTermNode?.TermName;
+            var defaultOperator = defaultTermNode?.Operation;
+            var shouldRestoreDefault = false;
+
+            if (defaultTermNode is not null)
+            {
+                var value = defaultTermNode.ToString();
+                if (!_operators.Any(opt => value.Contains(opt, StringComparison.Ordinal)))
+                {
+                    // Use an unary operator based on a full quoted string.
+                    defaultOperator = new UnaryNode(value, OperateNodeQuotes.Double);
+                }
+
+                var selectedContentType = GetSelectedContentType(model);
+
+                if (selectedContentType != null)
+                {
+                    defaultTermName = GetDefaultTermName(selectedContentType);
+                }
+
+                if (defaultTermName != defaultTermNode.TermName || defaultOperator != defaultTermNode.Operation)
+                {
+                    shouldRestoreDefault = true;
+
+                    model.FilterResult.TryRemove(defaultTermNode.TermName);
+                    model.FilterResult.TryAddOrReplace(new DefaultTermNode(defaultTermName, defaultOperator));
+                }
+            }
+
+            // Because admin filters can add a different index to the query this must be added as a Query<ContentItem>().
             var query = _session.Query<ContentItem>();
 
             query = await model.FilterResult.ExecuteAsync(new ContentQueryContext(_serviceProvider, query));
 
             // After the q=xx filters have been applied, allow the secondary filter providers to also parse other values for filtering.
-            await _contentsAdminListFilters.InvokeAsync((filter, model, query, updater) => filter.FilterAsync(model, query, updater), model, query, updater, _logger);
+            await _contentsAdminListFilters
+                .InvokeAsync((filter, model, query, updater) => filter.FilterAsync(model, query, updater), model, query, updater, _logger);
+
+            if (shouldRestoreDefault)
```

Comment:
Do we really need to restore the default?

Reply:
I think we do since we swap `text` with a custom value. But not sure other you had it done in your version :)

Reply:
> But not sure other you had it done in your version :)

Oups ;) Hmm I think we don't need it now that we are using options, worth to try

Reply:
I think we need it to remove the added double quotes. But something isn't right because when we add the quote using the UnaryNode it does not change the behavior search. I'll look more at it tomorrow

Reply:
Just tried, yes we need to remove the added quotes otherwise they are then rendered.

I just tried the quoted string option (without any default term name per content type), it works on my side.

Reply:
@jtkech 
Feel free to commit any changes you see fit. Maybe I am missing the idea behind the double quotes.